src/components/VerifyTooltip.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import Container from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+
+import Badge from '@material-ui/icons/PersonPin';
+import AccessTime from '@material-ui/icons/DateRange';
+import Note from '@material-ui/icons/Note';
+import Person from '@material-ui/icons/Person';
+import Nature from '@material-ui/icons/Nature';
+
+const VerifyTooltip = ({ capture, showCaptureClick }) => {
```

Comment:
I'd suggest renaming this component `CaptureDetailTooltip` since it will be used in a few different tools as a quick-look for the capture details.